.github/workflows/lagom-template.yml

```diff
@@ -190,7 +190,7 @@ jobs:
         if: ${{ always() && inputs.fuzzer == 'NO_FUZZ' }}
         uses: actions/upload-artifact@v4
         with:
-          name: libweb-test-artifacts-${{ inputs.os_name }}
+          name: libweb-test-artifacts-${{ inputs.os_name }}-${{inputs.os}}-${{inputs.fuzzer}}-${{inputs.toolchain}}-${{inputs.clang-plugins}}
```

Comment:
nit: `inputs.os` should be redundant with `os_name`

Reply:
Yeah I wasn't sure about that, so I put it in just in case. I can take it out though.